packages/tasks_repository/tasks_data.dart

```diff
@@ -0,0 +1,30 @@
+class Task {
```

Comment:
@kamilgadek You already have a `Task` model [here](https://github.com/kamilgadek/to_do_list_kamilg/blob/main/packages/tasks_repository/lib/src/models/task.dart)
So there is no need to create it again. Please remember that the code should be **DRY** - Do Not Repeat Yourself.

What to do:
- remove lines 1 - 12 (we already have a Task model)
- adjust the tasks to a model you already have 
- create a new folder inside `tasks_repository/lib/src` named `data` and store this file there.

Let me know if any difficulties.

Reply:
Ok, I made the changes. Should I do a git pull in this pull request now?